book/website/collaboration/remote-collab/remote-collab-code-review.md

```diff
@@ -0,0 +1,34 @@
+# Collaborative Code Review
+
+## 1. Problem statement
+* Ensuring your code to be reviewed is accessible/reproducible (across different operating systems and coding setups)
+* Trying to learn a new language and not knowing the most efficient way to solve a given problem. Others might have better ways to solve that problem which you might not know about. 
+* Getting feedback about our code's interface, in order to make it more usable.
+* How do you get this feedback while being considerate of other people's time?
+* Getting started and gauging interest in a group code review
+
+## 2. Solutions & Recommendations
```

Comment:
This heading is not clearly related to the content that follows - suggest this is changed to "best practices for preparing for code review" or similar

It might also be useful to break this down into further subheadings to split out good coding practices before a review is requested, and good practices for organising a review